modules/cluster-logging-collector-external.adoc

```diff
@@ -134,16 +208,22 @@ For example:
   </store>
 ----
 
+. Create a ConfigMap named `secure-forward` in the `openshift-logging` namespace from the configuration file:
++
+----
+$ oc create configmap secure-forward --from-file=secure-forward.conf.yaml  -n openshift-logging
+----
+
 . Optionally, create a secret named `secure-forward` in the `openshift-logging` namespace.
 +
 ----
-$ oc create secret secure-forward --from-file=<arbitrary-name-of-key>=<file-name-of-ssl-private-key> --from-file=<arbitrary-name-of-cert>=<file-name-of-ssl-cert>
+$ oc create secret generic secure-forward --from-file=<arbitrary-name-of-key>=<file-name-of-ssl-private-key> --from-file=<arbitrary-name-of-cert>=<file-name-of-ssl-cert>
```

Comment:
1. arbitrary-name-of-key1, Yes.  but the name will be used in the secury-forward configmap.
2. How about only import necessary secrects in the example:

oc create secret generic secure-forward --from-file=arbitrary-name-of-key1=cert_file_from_fluend_receiver --from-literal=shared_key=value_from_fluend_receiver